config/urls.py

```diff
@@ -328,7 +329,9 @@
         ),
         path(
             "review/",
-            TemplateView.as_view(template_name="review/review_process.html"),
+            RedirectView.as_view(
+                url="/doc/formal-reviews/submissions.html", permanent=True
```

Comment:
I'd prefer to use a _slightly_ more dynamic url here.

```suggestion
                url=reverse_lazy("docs-user-guide",  content_path="formal-reviews/submissions.html"), permanent=True
```